### DISCUSSION TITLE
New Atom stack laser modules

### DISCUSSION THREAD

**Discussion by dobby117** [arkypita/LaserGRBL#1968] [General]
Can anyone help with adding the new atom stack laser modules into the devices. I have the 100w version and the new fiber laser module coming; however I have no idea how to add them into the device database. Please help

**Comment by arkypita** [comment]
Go to menu "Grbl", "Material DB", scroll down to last line and add your entries

![image](https://user-images.githubusercontent.com/8782035/206226163-474656f5-9285-4d5e-9609-84843a5596af.png)

**Reply by dobby117** [reply]
Thank you
